### DISCUSSION TITLE
ButtonLink danger

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1162] [Proposals]
### Goal

To be able to have a destructive buttonLink ( in danger-red)

### Context

The current buttonLink options doesn't allow links to be in danger. 

### Impact

Allow destructive actions with buttonLinks

### Discussion Questions

Why is not supported yet? Is there a reason for not allowing a destructive action in a buttonLink?

**Comment by yceballost** [comment]
Already exist a discussion on this topic, can you share your opinion here? :)
https://github.com/Telefonica/mistica-design/discussions/501
